article/_posts/2019-09-11-state-regularized-rnn.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: review
+title: "State-Regularized Recurrent Neural Networks"
+tags: rnn layers 
```

Comment:
The tags file contains the uppercase version of `rnn`. Not sure whether tagging is case sensitive. If yes, and you configured the pre-commit hook and this passed, then the pre-commit may need to be changed as well.